scripts/deploy.sh

```
@@ -87,19 +107,19 @@ if [ "$SUCCESS" != "true" ]; then
-# 9. nginx 포트 전환
+# 10. nginx 포트 전환
 echo "nginx 전환 중..."
 sudo sed -i "s/set \$service_port .*/set \$service_port $NEW_PORT;/" $NGINX_CONF
 sudo nginx -t
 sudo systemctl reload nginx
 echo "nginx 전환 완료"
 
-# 10. 기존 컨테이너 종료
+# 11. 기존 컨테이너 종료
 echo "기존 컨테이너 종료 (${CURRENT_PORT})"
 docker stop ongi-backend-${CURRENT_PORT} 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)